README.md

```diff
@@ -3,26 +3,43 @@
 [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/232a8tadrrn8jv0k/branch/master?svg=true)](https://ci.appveyor.com/project/cgutman/moonlight-android/branch/master)
 [![Translation Status](https://hosted.weblate.org/widgets/moonlight/-/moonlight-android/svg-badge.svg)](https://hosted.weblate.org/projects/moonlight/moonlight-android/)
 
-[Moonlight for Android](https://moonlight-stream.org) is an open source client for NVIDIA GameStream and [Sunshine](https://github.com/LizardByte/Sunshine).
+[Moonlight](https://moonlight-stream.org/) for Android is an open source client for
+[Nvidia GameStream](https://www.nvidia.com/en-us/support/gamestream/) and
+[Sunshine](https://github.com/LizardByte/Sunshine).
 
-Moonlight for Android will allow you to stream your full collection of games from your Windows PC to your Android device,
-whether in your own home or over the internet.
+Moonlight for Android will allow you to stream your full collection of games from your computer to
+your Android device, whether in your own home or over the internet.
 
-Moonlight also has a [PC client](https://github.com/moonlight-stream/moonlight-qt) and [iOS/tvOS client](https://github.com/moonlight-stream/moonlight-ios).
+Moonlight also has clients for [PC](https://github.com/moonlight-stream/moonlight-qt),
+[iOS/tvOS](https://github.com/moonlight-stream/moonlight-ios), and
+[more](https://moonlight-stream.org/).
 
-You can follow development on our [Discord server](https://moonlight-stream.org/discord) and help translate Moonlight into your language on [Weblate](https://hosted.weblate.org/projects/moonlight/moonlight-android/).
+You can follow development on our [Discord server](https://moonlight-stream.org/discord)
+and help translate Moonlight into your language on
+[Weblate](https://hosted.weblate.org/projects/moonlight/moonlight-android/).
```

Comment:
I have one question about translations (for developers) that could probably be clarified here.

https://github.com/moonlight-stream/moonlight-android/pull/1314#discussion_r1691817532